reactions.json

```diff
@@ -208,6 +208,34 @@
     "KI": "iodobenzene",
     "HBF4 / (Heat)": "fluorobenzene"
   },
+  "but-2-ene": {
+    "H2O / dil H2SO4": "butan-2-ol",
+    "PCC": "butanone"

```

Comment:
Are you sure this is correct?

Reply:
Yes, I am sure. But if that seems doubtful Ill go with more popular reagent for oxidation KMnO4 then!